### PR TITLE
parent the CSS for search-input-form in addition to typeahead-dropdown

### PR DESCRIPTION
**Pull request by bigglesandginger**

### CODE REVIEW

**Comment by bigglesandginger** [comment]
@magskai In that specific case - No there is no parent element called `typeahead-dropdown`.  I would place the style in `advanced-search-input.scss` but how do I do that without copy/pasting everything that is under that parentage?  I do not see how scss allows for that, I only see how to make a variable like `15px` not a whole slew of style info - but I am new to it.

**Comment by magskai** [comment]
@bigglesandginger Oh I see. There are some typeahead components in TF that haven't been migrated into the `typeahead-dropdown` element. That would be a big annoying change for right now though. 

Here's what I would do:
Since all of our typeahead dropdowns should share the same base style, remove `typeahead-dropdown` from the hierarchies in `typeahead-dropdown.scss`, so now those styles apply to all `.twitter-typeahead`.

Double-check the views that have typeahead without `typeahead-dropdown`, such as the Search Bar, Organizations Form View, etc. and make sure that everything still looks right. Some views may require overrides. For example, the Add Users dropdown in Organizations Form View is now too wide and needs `.tt-menu` width set to `fit-content` in `view/organizations/form.scss`. There's also too wide of a margin between `.tt-input` and `.tt-menu` in that view so you'll have to address that too.

**Comment by chris-dickson** [comment]
@magskai, can you take a look at the changes @bigglesandginger pushed?

File: tellfinder-ui-core.iml [hunk 1]
```diff
@@ -0,0 +1 @@
+    <output-test url="file://$MODULE_DIR$/build" />
```
[the remark this answers is not in the document]
Comment (bigglesandginger) [reply tellfinder-ui-core.iml]: Take a look now - and I have no idea what that IML file is or what it is for

**Pull request merged**